calificativ.js

```diff
@@ -13,4 +13,23 @@
      3: IS
      2: IS
      1: IS
-*/
\ No newline at end of file
+*/
+function calificative(a){
+    let note=[1,2,3,4,5,6,7,8,9,10];
+    a=note;
+    let cal;
+    for (let i = 0; i < arguments.length; i++) {
```

Comment:
Funcția ta face ce trebuie, dar gândește-te foarte bine la ce face fiecare instrucțiune. Sunt multe la care ai putea să renunți.

Reply:
Am lasat asa. 
function calificative(a){
  let cal;
  for (let i = 0; i < arguments.length; i++)

Reply:
function calificative(a){
  let cal;
   
    if (a > 8 && a <= 10) {
      cal = "FB";
    } else if (a > 6 && a <= 8) {
      cal = "B";
    } else if (a > 4 && a <= 6) {
      cal = "S";
    } else if (a<= 4 && a> 0) {
      cal = "IS";
    }
    return cal;
   
}

console.log(calificative(4));

Lol :)) chiar era mai simplu decat anticipasem

Reply:
Să faci te rog push, să vedem forma finală